src/feed/tests/test_feed_cache.py

```diff
@@ -150,29 +150,30 @@ def test_invalidate_feed_cache_calls_delete_for_expected_keys(self, mock_cache):
 
 
 class FeedCachingTests(APITestCase):
-    def setUp(self):
-        cache.clear()
-        self.user = create_random_default_user("cache_test_user")
-        self.hub, _ = Hub.objects.get_or_create(
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
This method would run once instead before every test to ensure we aren't recreating test data that is unlikely to change